apps/console/src/protected-app.tsx

```diff
@@ -94,7 +94,7 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
         state
     } = useAuthContext();
 
-    const dispatch: Dispatch<any> = useDispatch();
+    const dispatch: string = useDispatch();
```

Comment:
Is this intentional? shouldn't this be as follows?

```
import { Dispatch } from "redux"

const dispatch: Dispatch = useDispatch()
```

Reply:
Can you PRAA if everything looks fine. I've reverted the test.